doc/build-macos.md

```diff
@@ -0,0 +1,65 @@
+# Building dogecoin-qt 1.14 on MacOS #
+
+Tested on MacOs Ventura and Sonoma on Intel (x86_64) and Apple Silicon (arm64) macs.
+
+### Clone dogecoin locally, or check it out, etc. ###
+
+```sh
+git clone https://github.com/dogecoin/dogecoin.git
+```
+
+### Set up OSX basic build dependencies. ##
+
+Install xcode-select commandline utils.
+
+```sh
+xcode-select --install
+```
+
+Make sure frameworks dir is properly owned...
+
+```sh
+sudo mkdir -p /usr/local/Frameworks
+sudo chown $(whoami):admin /usr/local/Frameworks
+```
+
+Install Brew. (If you already have Brew installed, perform a `brew update`.)
+
+```sh
+git clone https://github.com/Homebrew/brew homebrew
+eval "$(homebrew/bin/brew shellenv)"
+brew update --force --quiet
+chmod -R go-w "$(brew --prefix)/share/zsh"
+```
+
+Install dependencies via Brew.
+
+```sh
+brew install autoconf automake libtool miniupnpc openssl pkg-config protobuf@21 \
```

Comment:
Suggest ```brew install -v autoconf ... ...``` since the process can appear to freeze for long periods of time without the verbose flag.